Wallix/wallix-bastion/tests/test_completed_event_event.json

```
@@ -0,0 +1,46 @@
+{
+  "input": {
+    "message": "2025-04-23T17:29:22+02:00 test rdpproxy 11292 - - [RDP Session] session_id=\"19662eb3c2c67400005000000000\" client_ip=\"1.16.11.1\" target_ip=\"1.1.11.1\" user=\"test@test.fr\" device=\"test_device\" service=\"RDP\" account=\"test@test.fr\" type=\"COMPLETED_PROCESS\" command_line=\"C:\\\\Windows\\\\System32\\\\testBroker.exe -Embedding\"",
+    "sekoiaio": {
+      "intake": {
+        "dialect": "WALLIX Bastion",
+        "dialect_uuid": "6b8cb346-6605-4240-ac15-3828627ba899"
+      }
+    }
+  },
+  "expected": {
+    "message": "2025-04-23T17:29:22+02:00 test rdpproxy 11292 - - [RDP Session] session_id=\"19662eb3c2c67400005000000000\" client_ip=\"1.16.11.1\" target_ip=\"1.1.11.1\" user=\"test@test.fr\" device=\"test_device\" service=\"RDP\" account=\"test@test.fr\" type=\"COMPLETED_PROCESS\" command_line=\"C:\\\\Windows\\\\System32\\\\testBroker.exe -Embedding\"",
+    "event": {
+      "action": "COMPLETED_PROCESS"
+    },
+    "destination": {
+      "address": "1.1.11.1",
+      "ip": "1.1.11.1"
+    },
+    "process": {
+      "command_line": "C:\\\\Windows\\\\System32\\\\testBroker.exe -Embedding"
```

Can you remove the double-backslahes?
```suggestion
      "command_line": "C:\\Windows\\System32\\testBroker.exe -Embedding"
```
see [this issue](https://github.com/SekoiaLab/integration/issues/550) and [this PR](https://github.com/SEKOIA-IO/intake-formats/pull/1605)